IOSDeviceLib/ServerHelper.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#ifdef _WIN32
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#else
+#include <netinet/in.h>
+#include <arpa/inet.h>
+#include "Declarations.h"
```

Comment:
`#include <netinet/in.h>
#include <arpa/inet.h>
#include "Declarations.h"`
Are in the `else` of the `#ifdef`. They will be included only for mac.